kivy/uix/camera.py

```diff
@@ -90,7 +90,9 @@ def __init__(self, **kwargs):
         fbind('resolution', on_index)
         on_index()
 
-    def on_tex(self, *l):
+    def on_tex(self, camera):
+        self.texture = texture = camera.texture
```

Comment:
We can still check and update only when texture is None or when camera.texture.size is not equal to self.texture_size, right ?

Reply:
@Cheaterman I'm perplexed at where this `tex` variable comes from. Is this from I couldn't use pycharm to tell me where `self.tex` is defined.